infrastructure/templates/warehouse.yaml

```diff
@@ -33,3 +37,52 @@ Resources:
     Properties:
       ManagedPolicyArns:
         - !Ref UploadToRawPolicy
+
+  # CloudTrail
+  CloudTrailLogsBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: nccid-data-warehouse-object-logs
```

Comment:
Just thinking after the fact, wouldn't it make sense for the bucket name (the main bucket and the logs as well) to be a variable, with a default value? That would allow testing the template easier, as bucket names are globally unique, so whatever is in production cannot be used for tests.

Reply:
Good call. On GCP, we append a suffix to unique names. Thus, for instance, we could allow the user to pass a suffix in the driver script (e.g. `staging` or `prod` or whatever). We then append that to everything name.

What do you think?

Reply:
I think that sounds like a good pattern, especially if we have experience with this mental model! Later we can always adjust things.

Reply:
Added in a72da24